prov/util/src/util_cntr.c

```diff
@@ -204,11 +204,55 @@ static struct fi_ops_cntr util_cntr_no_wait_ops = {
 	.wait = fi_no_cntr_wait,
 };
 
+static void util_peer_cntr_inc(struct fid_peer_cntr *cntr)
+{
+	struct util_cntr *util_cntr = cntr->fid.context;
+
+	util_cntr->cntr_fid.ops->add(&util_cntr->cntr_fid, 1);
+}
+
+static void util_peer_cntr_incerr(struct fid_peer_cntr *cntr)
+{
+	struct util_cntr *util_cntr = cntr->fid.context;
+
+	util_cntr->cntr_fid.ops->adderr(&util_cntr->cntr_fid, 1);
+}
+
+static struct fi_ops_cntr_owner util_peer_cntr_owner_ops = {
+	.size = sizeof(struct fi_ops_cntr_owner),
+	.inc = &util_peer_cntr_inc,
+	.incerr = &util_peer_cntr_incerr,
+};
+
+static uint64_t ofi_peer_cntr_read(struct fid_cntr *cntr_fid)
+{
+	struct util_cntr *cntr = container_of(cntr_fid, struct util_cntr, cntr_fid);
+
+	assert(cntr->cntr_fid.fid.fclass == FI_CLASS_CNTR);
```

Comment:
Forgot to remove?

Reply:
See this: Only the last 2 commits are related, the first 3 commits are from https://github.com/ofiwg/libfabric/pull/9117 and will be dropped after it's merged